### PR TITLE
[Owl] Add edge-to-edge support for Owl

### PR DESCRIPTION
**Pull request by JolandaVerhoef**

### CODE REVIEW

**Comment by Sydfg** [comment]
jv/owl-e2e

**Comment by Sydfg** [comment]
1. > 
[Uploading basic-android-kotlin-compose-birthday-card-app-main.zip…]()

File: Owl/app/src/main/res/values/color.xml [hunk 1]
```diff
@@ -13,8 +13,5 @@
   the License.
   -->
 <resources>
-
     <color name="immersive_sys_ui">#33000000</color>
```
Comment (nickbutcher) [inline Owl/app/src/main/res/values/color.xml:16]: It looks like this is only used by the theme you're removing in this PR. I think you can also remove this color / entire file?

Comment (JolandaVerhoef) [reply Owl/app/src/main/res/values/color.xml:16]: No, it's used as the scrim color in the enableEdgeToEdge method

**Review by arriolac — APPROVED** [review]

**Review by MagicalMeghan — APPROVED** [review]

**Pull request merged**